### PR TITLE
005 inscription enregistrement

### PR DESCRIPTION
**Pull request by ilalande**
Part 2 of login and checking : finishing connect and fixing bugs

### CODE REVIEW

**Review by ThomasDCW — DISMISSED** [review]
ok for me

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Stop `git add .` :tired_face:

File: dd [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  004-connexion-phase1-back[m
+  005-inscription-enregistrement[m
+  009-favoris-pagefav[m
+  011-favoris-consult[m
+  012-home-1.0[m
+  BackRoutes[m
+  LessonCard-LessonsPage-TopicPages-API-call-and-CSS-fix[m
+  addFavoris[m
+  auth[m
+  axios[m
```
[the remark this answers is not in the document]
Comment (ThomasDCW) [reply dd:15]: dunno ...

**Review by ThomasDCW — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**